### ISSUE TITLE
WHM 0.4.4 won't upload a specified sub-directory

### ISSUE THREAD

**Issue by theWebalyst** [maidsafe/safe_examples#344]
I created a public id and website with an earlier version of WHM, but using 0.4.4 I cannot upload a specified subdirectory.

Steps:
1. having deleted the files in a sub-directory of the www root (so I can re-upload them), the sub-directory is no longer listed. So I can't go into the directory and must instead choose "Upload Folder"
2. Click "+" > "Upload Folder", choose the folder, "Open". 

This doesn't upload the folder, but uploads its contents to the www root.

So I have no way to re-create the sub-directory when using WHM 0.4.4. To upload files into a sub-directory, that directory must already exist. I have confirmed this is NOT the case with WHM 0.4.0 so that has enabled me to re-create the subdirectory.

I don't think it matters, but I was using peruse with WHM 0.4.4 (but had to use SAFE Browser to test with WHM 0.4.0 because 0.4.4 won't auth with peruse).

**Comment by bochaco** [comment]
Thanks @theWebalyst  for reporting this, we'll look into it. Just to clarify, I think you cannot use WHM 0.4.0 with Peruse because of the incompatibility of auth reqs/responses when upgrading the safe_client_libs, Peruse is using the very latest version as SAFE browser v0.8.x.

**Comment by theWebalyst** [comment]
Yes, I discovered this. So I was using different browsers for compat with WHM 0.4.0 (I thought SB 0.7.0) and WHM 0.4.4 (peruse).
